### ISSUE TITLE
Water

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-two#21]
Living in _Northern_ Europe, Water Scarcity is not something we _think_ about very often, however it's a fact of life for much of the planet. see: https://en.wikipedia.org/wiki/Water_scarcity

Despite The Donald _denying_ Climate Change: https://twitter.com/realDonaldTrump/status/265895292191248385
![donald-trump-denies-climate-change](https://cloud.githubusercontent.com/assets/194400/20188892/af5199d6-a772-11e6-9435-9a3eb2736de4.png)

> There is **Scientific _Consensus_**: http://climate.nasa.gov/scientific-consensus/ that Climate Change _exists_ and Humans are contributing to it.

As the Earth's _average_ temperature rises the warmer regions of land will become dryer, so the water scarcity problem is going to get _worse_. 

# Water is our `#1` priority for `#Phase2` (_without it everything else will fail_)

+ [ ] Only consider land that has a _good_ (_preferably great_) source of water e.g: bordering or including a river, stream or spring
+ [ ] Understand from the historical weather data what the rainfall for the region is.
+ [ ] Does the land have an _existing_ water storage (eco)system e.g a Lake or reservoir?
+ [ ] If a well is the only source of water, how deep is it and what is the estimated remaining volume?

> Relates to: https://github.com/dwyl/phase-two/issues/16

**Comment by nelsonic** [comment]
The warming is _accelerating_:
https://www.washingtonpost.com/news/energy-environment/wp/2016/11/17/the-north-pole-is-an-insane-36-degrees-warmer-than-normal-as-winter-descends/

**Comment by c0rr314** [comment]
I agree that without water this type of project will struggle to get off the ground. A lake or reservoir should have a feeder too, like a river or natural spring, to keep replenishing it. A well/ borehole should be checked by a water diviner to ensure that there is sufficient ground water to sustain it.

**Comment by nelsonic** [comment]
I want to avoid a well/borehole and stick to being next to a year-round river.
did you see: https://github.com/dwyl/phase-two/issues/25 ? 😉

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/50239287-b3b00080-03b9-11e9-8699-e5cb143d99b2.png)
https://mashable.com/2016/03/21/water-access-cost

**Comment by nelsonic** [comment]
Pumped Dry: The Global Crisis of Vanishing Groundwater
[![image](https://user-images.githubusercontent.com/194400/60175799-7b77f780-980c-11e9-837e-62bbc6ba5b80.png)](https://youtu.be/RjsThobgq7Q)
https://youtu.be/RjsThobgq7Q

**Comment by nelsonic** [comment]
People in Madagascar earning min wage spend up to 45% of their income on obtaining fresh water: https://youtu.be/eABuoaf0Vbs

**Comment by nelsonic** [comment]
The fight for water | DW Documentary: https://youtu.be/1MZFrJPPIQ8

**Comment by nelsonic** [comment]
10 biggest world threats of 2023, ranked: https://youtu.be/-QvFkwyX28c
[![top 10 world risks](https://user-images.githubusercontent.com/194400/215020436-15bd64fb-5ff5-46b3-a62f-1d5c931e932e.png)](https://youtu.be/-QvFkwyX28c)

"Water is an ongoing realtime issue around the world".

**Comment by nelsonic** [comment]
Global Water Crisis: Why Michael Burry is Investing in Blue Gold: https://youtu.be/04jCSkbhsOg

**Comment by nelsonic** [comment]
The Surprising Solutions to the World's Water Crisis: https://youtu.be/FDY2McKLvlM

**Comment by carolrmc** [comment]
This is incredibly frightening. We know what day zero has been for our
families in. Cape Town, but sadly not much has been done to secure water
for the future by the present government. Not sure what they’re waiting
for!
Inspiring to see what has been done in Singapore; people with their back to
the wall have worked together to find a solution.  Hats off to them! May
the world take a leaf out of their book and follow their lead

**Comment by nelsonic** [comment]
Singapore gets many things right. But worth understanding _how_ ... 💭 
But on the topic of water, they definitely understood what needed to be done and did it. ✅ 
Most other countries have too much Gov bureaucracy and will not take _action_ in time. ⏳ 
Or in the worst cases will _talk_ about fixing the problem but then the money will be squandered. 🤦‍♂️
